pages.zh/common/git.md

```diff
@@ -0,0 +1,29 @@
+# git
+
+> 分布式版本控制系统.
+> 类似如 commit、add、branch、checkout、push 等子命令都有自己的使用文档，可以通过 tldr git {{子命令}} 的形式查阅.
```

Comment:
```suggestion
> 类似如 `commit`、`add`、`branch`、`checkout`、`push` 等子命令都有自己的使用文档，可以通过 `tldr git {{子命令}}` 的形式查阅.
```

Reply:
Thanks for the suggestion, I have already resolved.